src/OSPSuite.Infrastructure.Import/OSPSuite.Infrastructure.Import.csproj

```diff
@@ -32,7 +32,6 @@
     <PackageReference Include="NPOI" Version="2.5.4" />
     <PackageReference Include="OSPSuite.Utility" Version="4.0.0.4" />
     <PackageReference Include="LumenWorksCsvReader" Version="4.0.0" />
-    <PackageReference Include="System.Linq.Dynamic" Version="1.0.8" />
```

Comment:
@abdelr @georgeDaskalakis Not sure why this was required. This should probably be removed from develop as well